app/api/validations/user.py

```diff
@@ -173,8 +173,12 @@ def validate_new_password(data):
     if 'new_password' not in data:
         return messages.NEW_PASSWORD_FIELD_IS_MISSING
 
+    current_password = data['current_password']
     new_password = data['new_password']
 
+    if current_password in new_password:
```

Comment:
Hey, @IlyaMaier just a question here, are we checking that the new and old passwords do not match, or should the old password be not in any form match/be a subset of new password? 
In the case of the former, `==` sign would be better to use, else this is good.

Reply:
Because the msg says 
```
"New password should not be same"
 "with the current password."
```
but we are checking for similar instead of same.

Reply:
yup. You are right! That's my bad. I'll fix that now. And thank you for a review :)